src/Queue/RoadRunnerQueue.php

```
@@ -69,55 +71,40 @@ public function size($queue = null): int
     {
         $stats = $this->getStats($queue);
 
-        return $stats->getActive() + $stats->getDelayed();
+        return $stats->getActive() + $stats->getDelayed() + $stats->getReserved();
```

PHPStan (2.2.7)

[error] Binary operation "+" between int|string and int|string results in an error.


(binaryOp.invalid)